server/routes.js

```diff
@@ -629,6 +629,10 @@ exports.mountQuintypeAt = function (app, mountAt) {
 exports.ampRoutes = (app, opts = {}) => {
   const { ampStoryPageHandler, storyPageInfiniteScrollHandler } = require("./amp/handlers");
 
+  app.use((req, res, next) => {
+    res.header("Access-Control-Allow-Origin", req.headers.origin);
+    next();
+  });
```

Comment:
whitelisting all origins violates https://amp.dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests/